docs/README-developers.md

```diff
@@ -96,3 +96,92 @@ More details about tests can be found in the [tests folder](../tests/).
 
 *Note*: These run integration tests that invoke `c2rust transpile`.
 `cargo test` only runs unit tests and doc tests as of now.
+
+## Documentation
+
+Local documentation can be built with the normal `cargo doc` and `cargo doc --open`.
+
+### `rustc`
+
+However, as the `#![feature(rustc_private)]` crates
+are not normal dependencies managed through `cargo`,
+accessing the documentation for those is trickier.
+
+#### `rust-src`
+
+`rust-src` is the `rustup` component containing `rustc`'s source code.
+This is needed to view `rustc`'s source when developing.
+
+`rust-src` is included in our [`rust-toolchain.toml`](../rust-toolchain.toml),
+so it will automatically be installed.  Otherwise,
+
+```sh
+rustup component add rust-src
+```
+
+will work.
+
+#### `rustc-docs`
+
+`rustc-docs` (not `rust-docs`) is the `rustup` component
+containing the `rustdoc` docs for `rustc`.
```

Comment:
Probably worth adding a link to https://rust-lang.github.io/rustup-components-history/ which says which components are available for each nightly. This is relevant for picking a good new nightly version when rolling the toolchain forward

Reply:
That's a good idea.  I'll add that link.  When upgrading the toolchain, though, it's generally easiest just to try a version and see if `rustup`/`cargo` complains.

Reply:
Fixed in ab1e7a38d664a8a3fdd261f77325a7b284620861.